src/Socket/socket.ts

```diff
@@ -777,5 +777,3 @@ function mapWebSocketError(handler: (err: Error) => void) {
 		)
 	}
 }
-
-export type Socket = ReturnType<typeof makeSocket>
```

Comment:
this export is not used. The same export already exists in `src/index.ts` -> export type WASocket = ReturnType<typeof makeWASocket>